benchmarks/benchmark_backends.py

```diff
@@ -1,27 +1,28 @@
-import sparse
-
 import numpy as np
 
-from .utils import SkipNotImplemented
+from .utils import SkipNotImplemented, skip_if_finch
 
 TIMEOUT: float = 500.0
-BACKEND: sparse.BackendType = sparse.backend_var.get()
 
 
 class Tensordot:
     timeout = TIMEOUT
 
     def setup(self):
+        skip_if_finch()
+
+        import sparse
+
         rng = np.random.default_rng(0)
 
         random_kwargs = {"density": 0.01, "random_state": rng}
-        if sparse.BackendType.Numba == BACKEND:
+        if sparse.BackendType.Numba == sparse.backend:
```

Comment:
Can we remove the `if` here as we support just one backend here?

Reply:
Done! Actually I removed whole `benchmark_backends.py` as Numba is benchmarked in other files and Finch will be run separately outside of asv.